constrain/library/G36CoolingOnlyTerminalBoxDeadbandAirflowSetpoint.py

```diff
@@ -1,25 +1,38 @@
 """
-G36 2021
 ### Description
 
 Section 5.5.5.2
-
 - When the Zone State is deadband, the active airflow setpoint shall be the minimum endpoint.
 
-Verification Item:
+### Code requirement
+
+- Code Name: ASHRAE Guideline 36
+- Code Year: 2021
+- Code Section: 5.5.5 Terminal Box Airflow Control
+- Code Subsection: 5.5.5.2 Deadband Airflow Control
+
+### Verification Approach
+
+The verification checks that when the zone is in deadband mode, the active airflow setpoint equals the minimum value within a specified tolerance. The minimum value varies depending on whether the system is in occupied mode or other modes (cooldown/setup/warmup/setback/unoccupied).
```

Comment:
where does the second sentence interpretation come from?

Reply:
The second interpretation comes from lines 85-88. Does this not make sense? Please let me know if you have a specific language to describe for this verification item.